src/components/ChannelPreview/ChannelPreview.js

```diff
@@ -4,24 +4,29 @@ import PropTypes from 'prop-types';
 import { ChatContext } from '../../context';
 import { useLatestMessagePreview } from './hooks/useLatestMessagePreview';
 
-const ChannelPreviewWithContext = React.memo((props) => {
```

Comment:
@DanC5 @vir I honestly don't have strong enough reason to remove this. But just with the experience of working on this sdk, I can say that we should handle this separately once all the parent components are converted as well. Re-renders are always tricky and so much dependence on how the end-user is using our components. So I will say lets handle this separately.

Reply:
was this causing issues?

Reply:
Nope!! Thats why `don't have strong enough reason ` haha ... but currently if something changes in ChatContext then ChannelPreview was getting re-rendered regardless ... thats what I noticed. So `ChannelPreviewWithContext` is not much helpful right now!! So thats why it would be better to handle it once parent components are ready as well